xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -90,8 +98,16 @@
         <param name="consumed">Marks the extent of the data that has been successfully processed.</param>
         <param name="examined">Marks the extent of the data that has been read and examined.</param>
         <summary>Moves forward the pipeline's read cursor to after the consumed data.</summary>
```

Comment:
Not related to your PR but these two overloads have the same exact description. Hard to choose which one to use.

Reply:
You're right. Let me modify the summary.

Reply:
```suggestion
        <summary>Moves forward the pipeline's read cursor to after the consumed data, marking the data as processed, read and examined.</summary>
```

@anurse @halter73 @davidfowl @jkotalik would you agree with my suggested new summary to differentiate it from the other overload?
I'm not entirely sure what the exact difference is between the `consumed` and the `examined` parameters, so I'm basing my summary on the param descriptions. Feel free to suggest an alternative if you can come up with a better description.

Reply:
Same question about the overload above this one (with only one param).